0000-interledger-protocol-4/0000-interledger-protocol-4.md

```diff
@@ -19,44 +19,58 @@ Interledger is a protocol for sending packets of money across different payment
+- **Condition and Fulfillment** - ILP packets are secured with conditions so that they cannot be lost or stolen by intermediary connectors. ILP Prepare packets are sent with a condition or SHA-256 hash in them. When the receiver receives the packet, they present the fulfillment, or valid preimage of the hash, to execute the transfer. Unlike in earlier versions of ILP, the conditions and fulfillments are NOT assumed to be enforced by the underlying ledger but they help peers reconcile their account balances. Two peers should only consider an ILP packet executed (and therefore affect their account balances) if 
 - **ILP Packets** - ILPv4 has three packet types: Prepare, Fulfill, and Reject, which roughly correspond to request, response, and error messages. Connectors forward Prepare packets from senders to receivers and the connectors relay the Fulfill or Reject packets back from the receivers to the senders.
-- **Ledger** - A system that tracks and transfers value between accounts. In earlier versions of Interledger, ILP packets were transmitted through transfers on ledgers. In ILPv4, ledgers are used primarily to settle obligations between connectors, their peers, and their customers. 
-- **Payment Channel** - A method of using signed claims against funds held on a ledger, rather than on-ledger transfers that may be too slow or expensive, to settle obligations. The channels used in ILPv4 may be unidirectional or bidirectional but generally they will be [simple](../0022-hashed-timelock-agreements/0022-hashed-timelock-agreements.md#simple-payment-channels) rather than [conditional](../0022-hashed-timelock-agreements/0022-hashed-timelock-agreements.md#conditional-payment-channels-with-htlcs) channels.
+- **Ledger** - A generic term for any system that tracks transfer of value between, and balances on, accounts. In earlier versions of Interledger, ILP packets were transmitted through transfers on ledgers. In ILPv4, ledgers are used primarily to adjust balances between nodes, to rebalance their accounts, following the exchange of one or more ILP Packets between the nodes. 
+- **Node** - A participant in an Interledger payment (node on the Interledger network), that has accounts with one or more other nodes. There are three specializations of a node:
+  * **Sender** - The party that originates payment
+  * **Receiver** - The final recipient of a payment
+  * **Connector** - The intermediaries between a sender and a receiver that forward ILP Packets. They may generate revenue from spreads on currency conversion, through subscription fees, or other means.
+- **Payment** - In the context of this specification, a payment is understood to mean the transfer of value from the sender (payer) to the receiver (payee). This may be affected by sending multiple ILP Packets that together result in the total transfer of value required to complete the payment.
```

Comment:
> This may be affected by sending multiple ILP Packets that together result in the total transfer of value required to complete the payment.

In this context, "affected" is the wrong verb. @adrianhopebailie's original "effected" was correct, if [confusing](https://xkcd.com/326/). A less tricky rephrase might be as follows:

> Higher-level protocols may execute a "payment" by sending a series of ILP Packets whose sum is equal to the desired payment value.